### PR TITLE
Add PyCon US 2017 talks and tutorials

### PR DESCRIPTION
**Pull request by redapple**
Fixes https://github.com/pyvideo/data/issues/245

### CODE REVIEW

File: pycon-us-2017/videos/kelsey-hightower-keynote-pycon-2017.json [hunk 1]
```diff
@@ -0,0 +1,16 @@
+{
+  "description": "\"Speaker: Kelsey Hightower\n\nKeynote\n\nSlides can be found at: https://speakerdeck.com/pycon2017 and https://github.com/PyCon/2017-slides\"",
+  "duration": 2004,
+  "recorded": "2017-05-21",
+  "speakers": [
+    "Kelsey Hightower"
+  ],
+  "thumbnail_url": "https://i.ytimg.com/vi/u_iAXzy3xBA/hqdefault.jpg",
+  "title": "Keynote: Kelsey Hightower",
```
Comment (Daniel-at-github) [inline pycon-us-2017/videos/kelsey-hightower-keynote-pycon-2017.json]: Title: kubernetes for Pythonistas (time 0:39)

File: pycon-us-2017/videos/kelsey-hightower-keynote-pycon-2017.json [hunk 2]
```diff
@@ -0,0 +1,16 @@
+{
+  "description": "\"Speaker: Kelsey Hightower\n\nKeynote\n\nSlides can be found at: https://speakerdeck.com/pycon2017 and https://github.com/PyCon/2017-slides\"",
```
Comment (Daniel-at-github) [inline pycon-us-2017/videos/kelsey-hightower-keynote-pycon-2017.json]: The \" seems a little strange surrounding the description

**Comment by redapple** [comment]
Thanks for the review @Daniel-at-github .
I fixed the keynote title and 2 descriptions surrounded by `"` (were what YouTube had as description)

File: pycon-us-2017/videos/christopher-fonnesbeck-probabilistic-programming-with-pymc3-pycon-2017.json [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+  "duration": 1863,
+  "recorded": "2017-05-21",
+  "speakers": [
+    "Christopher Fonnesbeck"
+  ],
+  "thumbnail_url": "https://i.ytimg.com/vi/5TyvJ6jXHYE/hqdefault.jpg",
+  "title": "Probabilistic Programming with PyMC3",
```
Comment (Daniel-at-github) [inline pycon-us-2017/videos/christopher-fonnesbeck-probabilistic-programming-with-pymc3-pycon-2017.json]: Title: The memory chronicles: A tale of two Pythons (Seen in the thumbnail).

**Comment by redapple** [comment]
The thumbnail is showing the wrong slide it seems, because the talk is
indeed about pymc3. And "the memory chronicle" is also added as another
talk.

Le 22 mai 2017 11:43 PM, "Daniel-at-github" <notifications@github.com> a
écrit :

*@Daniel-at-github* commented on this pull request.
------------------------------

In pycon-us-2017/videos/christopher-fonnesbeck-probabilistic-programming-
with-pymc3-pycon-2017.json
<https://github.com/pyvideo/data/pull/246#discussion_r117852768>:

> @@ -0,0 +1,16 @@
+{
+  "description": "Bayesian statistics offers robust and flexible
methods for data analysis\nthat, because they are based on probability
models, have the added\nbenefit of being readily interpretable by
non-statisticians. Until\nrecently, however, the implementation of
Bayesian models has been\nprohibitively complex for use by most
analysts. But, the advent of\nprobabilistic programming has served to
abstract the complexity of\nBayesian statistics, making such methods
more broadly available. PyMC3\nis a open-source Python module for
probabilistic programming that\nimplements several modern,
computationally-intensive statistical\nalgorithms for fitting Bayesian
models, including Hamiltonian Monte\nCarlo (HMC) and variational
inference. PyMC3\u2019s intuitive syntax is\nhelpful for new users,
and the reliance on Theano for much of the\ncomputational work has
allowed developers to keep the code base simple,\nmaking it easy to
extend the software to meet analytic needs. PyMC3\nitself extends
Python's powerful \"scientific stack\" of development\ntools, which
provide fast and efficient data structures, parallel\nprocessing, and
interfaces for describing statistical models.\n",
+  "duration": 1863,
+  "recorded": "2017-05-21",
+  "speakers": [
+    "Christopher Fonnesbeck"
+  ],
+  "thumbnail_url": "https://i.ytimg.com/vi/5TyvJ6jXHYE/hqdefault.jpg",
+  "title": "Probabilistic Programming with PyMC3",

Title: The memory chronicles: A tale of two Pythons (Seen in the thumbnail).

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub
<https://github.com/pyvideo/data/pull/246#pullrequestreview-39593818>, or mute
the thread

File: pycon-us-2017/videos/dave-forgac-share-your-code-python-packaging-without-complication-pycon-2017.json [hunk 4]
```diff
@@ -0,0 +1,16 @@
+{
+  "description": "If you want people to use your code you should package it! You may have\nheard that packaging is hard but the Python packaging ecosystem has\nevolved a lot over the years. Taking your beautiful code and sharing it\nwith the world is complex but it doesn't have to be complicated. In this\ntalk you will learn how to take advantage of modern tooling and\npractices so you can get boring stuff out of the way, publish quickly\nand frequently, and focus on your code.\n\nThis talk will cover:\n\n-  A (brief) history of Python packaging\n-  Python Packaging User Guide recommendations\n-  Distribution formats\n-  Anatomy of a package\n-  Automating package creation\n-  Adding:\n-  Testing\n-  CI\n-  Documentation\n-  Testing package installation\n-  Releasing to PyPI\n\nThis talk is for you if you're new to Python packaging and would like to\nlearn how to share you code or if you've worked with Python for a while\nand just aren't up-to-date with the latest packaging practices.\n",
```
Comment (Daniel-at-github) [inline pycon-us-2017/videos/dave-forgac-share-your-code-python-packaging-without-complication-pycon-2017.json]: Missing indentation:
~~~ json
  "description": "If you want people to use your code you should package it! You may have\nheard that packaging is hard but the Python packaging ecosystem has\nevolved a lot over the years. Taking your beautiful code and sharing it\nwith the world is complex but it doesn't have to be complicated. In this\ntalk you will learn how to take advantage of modern tooling and\npractices so you can get boring stuff out of the way, publish quickly\nand frequently, and focus on your code.\n\nThis talk will cover:\n\n-  A (brief) history of Python packaging\n-  Python Packaging User Guide recommendations\n-  Distribution formats\n-  Anatomy of a package\n-  Automating package creation\n-  Adding:\n    -  Testing\n    -  CI\n    -  Documentation\n-  Testing package installation\n-  Releasing to PyPI\n\nThis talk is for you if you're new to Python packaging and would like to\nlearn how to share you code or if you've worked with Python for a while\nand just aren't up-to-date with the latest packaging practices.\n",
~~~

File: pycon-us-2017/videos/russell-keith-magee-how-to-write-a-python-transpiler-pycon-2017.json [hunk 5]
```diff
@@ -0,0 +1,16 @@
+{
+  "description": "We all know Python is a powerful and expressive programming language.\nWhat you may not know is how much of the internals of Python itself is\nexposed for you to use and manipulate.\n\nIn this talk, you'll be introduced to the tools and libraries Python\nprovides to manipulate the compilation and execution of Python code. You\nwill also see how you can use those tools to target execution\nenvironments other than the CPython virtual machine.\n",
+  "duration": 1843,
+  "recorded": "2017-05-20",
+  "speakers": [
+    "Russell Keith-Magee"
```
Comment (Daniel-at-github) [inline pycon-us-2017/videos/russell-keith-magee-how-to-write-a-python-transpiler-pycon-2017.json]: Dr. Russell Keith-Magee

**Pull request merged**